src/app/components/Book/index.tsx

```diff
@@ -8,21 +8,24 @@ interface Props {
   imageUrl: string | null;
   title: string;
   author: string;
+  onPress: (event: GestureResponderEvent) => void;
 }
 
-function Book({ imageUrl, title, author }: Props) {
+function Book({ imageUrl, title, author, onPress }: Props) {
   return (
-    <View style={styles.bookContainer}>
-      <Image
-        style={styles.bookImage}
-        resizeMode="contain"
-        source={imageUrl ? { uri: imageUrl } : placeholderBookImg}
-      />
-      <View style={styles.decriptionContainer}>
-        <Text style={styles.title}>{title}</Text>
-        <Text style={styles.author}>{author}</Text>
-      </View>
-    </View>
+    <TouchableHighlight style={styles.bookContainer} onPress={onPress}>
+      <>
```

Comment:
Remove the unnecessary Fragment (<></>)

Reply:
The TouchableHighligh component, should has a child only? 🤔